index.rst

```diff
@@ -176,18 +152,73 @@ If working with an individual CSC, which as an operator would be a rare occurren
 
-.. Or the LOVE interface.
+
+If these types of tasks are performed from the LOVE interface, then the same result occurs where the defaults are loaded automatically.
 
 .. TODO: Add example on how to launch script from LOVE interface
 
 
-.. _section-configuration-interation_non_default:
+Discovering Available Override Configurations
+---------------------------------------------
+
+The easiest way to get information from a CSC programmatically is by using a Jupyter notebook server.
+From a notebook, observers, developers and power users can easily interact with the system through Python.
+
+In order to know which configurations are available for a specific CSC, users can read the ``configurationsAvailable`` event.
+This is done using a ``salobj.Remote`` class.
+
+.. code-block:: python
+
+    from lsst.ts import salobj
+
+    domain = salobj.Domain()
+
+    atdome = salobj.Remote(domain, "ATDome")
+
+    await atdome.start_task()
+
+    config_available = await atdome.evt_configurationsAvailable.aget(timeout=5.)
+
+    # This will print the available filenames.
+    print(config_available.configurations)
+
+    # This will print the git hash of the loaded configuration repository
+    print(config_available.versions)
+
+Often, this can also be accomplished using a high-level class that is designed to interact with a group of CSCs.
+
+.. code-block:: python
+
+    from lsst.ts.observatory.control import ATCS
+
+    atcs = ATCS()
+
+    await atcs.start_task
+
+    config_available = await atcs.rem.atdome.evt_configurationsAvailable.aget(timeout=5.)
+
+    # This will print the available filenames.
+    print(config_available.configurations)
+
+    # This will print the git hash of the loaded configuration repository
+    print(config_available.versions)
+
+It is also possible to check this information by querying the EFD or through the CSC summary information interface on LOVE.
+Examples of how to do this using the LOVE interface will be added when the functionality is ready.
+
+.. TODO: Add example of how to get this information from the EFD and LOVE.
+
 
-Selecting a non-default configuration
+.. _section-configuration-interaction_non_default:
+
+Selecting an Override Configuration
 -------------------------------------
 
 Selecting non-default configurations via control packages is also possible.
-A dictionary is used to override the appropriate configuration labels for each component that needs a non-default configuration.
+These are generally used for circumstances where customization is required, or a fallback from standard functionality is necessary.
+For example, if the new look-up tables, which are loaded by default, in the ATAOS are causing problems then we can use this procedure to overwrite the defaults by specifying a configuration file that contains the values from the previous look-up table.
+
+A dictionary is used to override the appropriate configuration override files for each component that needs a non-default configuration.
```

Comment:
Consider;

```
A dictionary is used to select the appropriate...
```

Reply:
good catch, but I went with  "A dictionary is used to provide the appropriate configuration override files..."